src/features/Students/StudentsPage/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useMemo } from 'react';
+import { useStudentEnrollments } from 'features/Students/data/slices';
+import { fetchStudentEnrollments } from 'features/Students/data/thunks';
+
+import Container from '@edx/paragon/dist/Container';
+import { getColumns } from 'features/Students/StudentsTable/columns';
+import { StudentsTable } from 'features/Students/StudentsTable/index';
+
+import { Pagination } from '@edx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+
+const StudentsPage = () => {
+  const { state, dispatch } = useStudentEnrollments();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        await fetchStudentEnrollments(dispatch);
+      } catch (error) {
+        logError(error);
+      }
```

Comment:
```suggestion
      };
```

Reply:
@anfbermudezme unncessery semicolon according to Lint.